src/main/java/io/github/wimdeblauwe/errorhandlingspringbootstarter/LoggingService.java

```diff
@@ -27,6 +32,20 @@ public void logException(ApiErrorResponse errorResponse, Throwable exception) {
         }
     }
 
+    public void logException(GraphQLError graphQLError, Throwable exception) {
```

Comment:
I would prefer a separate `GraphQlLoggingService` class. Common code could be moved into an abstract super class, or to helper components.

Reply:
We can extract the common logic into an AbstractLoggingService class. Then, each of the concrete implementations for REST and GraphQL can declare it's custom logic for `logException` method depending on the context. Is this what you had in mind?